uint8 sensor1
uint8 sensor2
uint8 sensor3
uint8 sensor4
uint8 sensor5
uint8 sensor6
uint8 sensor7
uint8 sensor8